### DISCUSSION TITLE
[v3] Alpine's $persist not working anymore in conjuction with @entangle

### DISCUSSION THREAD

**Discussion by lennardv2** [livewire/livewire#6271] [7. Bugs (archived)]
### Livewire version

3.0@beta

### Which PHP version are you using?

PHP 8.1

### Steps To Reproduce

Love the new livewire v3, thanks!

```
<div x-data="{ test: $persist(@entangle('test')) }">
    <input x-model="test" />
</div>
```

Same with
```
test: $persist(@entangle('test').live) 
```

The data is stored in LocalStorage when typing in the input, but upon page refresh it gets overridden by the livewire component. I need it to set the livewire prop upon refresh from the LocalStorage.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by hussain-alkhatib** [comment]
i have same issus

**Comment by PhiloNL** [comment]
Thanks for reporting! I've submitted a [failing test](https://github.com/livewire/livewire/pull/6756).

**Comment by bucari** [comment]
i have the same issue

**Comment by danielraab** [comment]
same here :/

**Reply by HugoGanoza** [reply]
same too

**Comment by boptom** [comment]
Anyone solved this, or have a work around? It's a killer feature which sadly no longer works.
